docs/pages/enroll-resources/mcp-access/getting-started.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: MCP Access Getting Started Guide
+description: Getting started with Teleport MCP access.
+---
+
+Teleport can provide secure connections to your MCP (Model Context Protocol)
+servers while improving both access control and visibility.
+
+This guides shows you how to:
+
+- Enroll the Teleport demo MCP server in your Teleport cluster.
+- Connect to the MCP server via Teleport.
+
+## How it works
+
+![How it works](../../../img/mcp-access/architecture-demo.svg)
+
+The Teleport Application Service includes a built-in demo MCP server designed to
+showcase how MCP access works.
+
+Users can configure their MCP clients such as Claude Desktop to start an MCP
+server using `tsh`. Once successfully authorized, `tsh` establishes a session
+with the Application Service.
+
+Once the session is established, the Application Service starts the in-memory
+demo MCP server. Teleport then proxies the connection between the client and the
+remote MCP server, applying additional role-based access controls such as
+filtering which tools are available to the user. While proxying, Teleport also
+logs MCP protocol requests as audit events, providing visibility into user
+activity.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+- A host, e.g., an EC2 instance, where you will run the Teleport Applications
+  Service.
```

Comment:
```suggestion
- A host (e.g., an EC2 instance) where you will run the Teleport Application Service.
```

Reply:
this is copied from other guides. it's used a few places. i will leave it is for now.